components/brave_wallet/common/config.gni

```diff
@@ -10,4 +10,6 @@ declare_args() {
   brave_infura_project_id = ""
   enable_bitcoin_by_default = !is_ios && !is_android
   enable_zcash_by_default = !is_ios && !is_android
+  enable_orchard_crate =
```

Comment:
I don't think this buildflag name makes sense here. We don't guard crates, we guard features. Also `enable_bitcoin_by_default` and `enable_zcash_by_default` should both drop `by_default` because it makes no sense in a  buildflag name.

Reply:
also `enable_bitcoin_by_default` and `enable_zcash_by_default` don't appear to be used anywhere anymore

Reply:
https://github.com/brave/brave-browser/issues/37599